source/testnet/guides/become-baker.rst

```diff
@@ -204,34 +172,48 @@ Provide a ``--no-restake`` flag to avoid automatically adding the
 rewards to the staked amount on the baker. This behavior is described on the
 section `Restaking the earnings`_.
 
-In order to start the node with these baker keys and start producing blocks the
-user first needs to shut down the current running node (either by pressing
+In order to start the node with these baker keys and start producing blocks you
+first need to shut down the current running node (either by pressing
 ``Ctrl + C`` on the terminal where the node is running or using the
 ``concordium-node-stop`` executable).
 
-After placing the file in the appropriate directory, start the
-node again using ``concordium-node``. The node will automatically start baking
-when the baker is included in the bakers for the current epoch. This will happen
-when finishing the epoch after the one in which the transaction for adding the
-baker was finalized.
+After placing the file in the appropriate directory (already done in the
+previous command when specifying the output file), start the node again using
+``concordium-node``. The node will automatically start baking when the baker
+gets included in the bakers for the current epoch.
+
+This change will be executed
+immediately and will take effect when finishing the epoch after the one in which
+the transaction for adding the baker was included in a block.
+
+.. table:: Time constraints when adding a baker
+
+   +------------------------------------+------------------------------------+----------------------+
+   |                                    | Transaction is included in a block | 2 epochs have passed |
```

Comment:
```suggestion
   |                                    | When transaction is included in a block | After 2 epochs |
```
I'm suggesting this to emphasize that these are moments in time

Reply:
I would also consider writing simply "When transaction is added" or "...included" or "...finalized" for brevity (here and below).

Reply:
Suggestion applied.